lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "command": {
+    "publish": { "ignoreChanges": ["*.md"] }
+  },
+  "packages": ["packages/*", "src/*"],
+  "version": "0.0.0"
+}
```

Comment:
What is the need of this file here. Can you double check if this is needed?

Reply:
Oh, I am so sorry. I think that happened because I always add files to commit using `git add .`, is that the reason?

I now realize how that is wrong in a bigger project like this one. From now on I will pay attention to which file to commit. I installed an VSCode AddOn to use the Conventional Commits you show me yesterday.

Reply:
All good. No stess

Reply:
Yes, it is a good habit indeed, thanks to pointing that out. I don't want to appear lazy or sloppy with my work, and now I understand it is on my side to check those kind of problems. I didn't know that this step was needed, but I will include it in my workflow.

Reply:
;) you fine dont worry about it. we are doing this for learning purposes